docs/performance/inserts/parallel.txt

```diff
@@ -0,0 +1,131 @@
+.. _parallel_inserts:
+
+================
+Parallel Inserts
+================
+
+Because CrateDB is a clustered data with every node capable of handling client
+requests, parallelizing (i.e. doing many at once) your inserts is one good way
+to increase insert performance.
+
+There are a few ways you can do this.
+
+.. rubric:: Table of Contents
+
+.. contents::
+   :local:
+
+Methods
+=======
+
+Use a Single Node
+-----------------
+
+Point your client application at a single node in your cluster, and send many
+concurrent requests.
+
+This is the least efficient way to do parallel inserts, because the
+resources on that node can easily become a bottleneck.
+
+Switch Between Multiple Nodes
+-----------------------------
+
+Configure your client application with a list of nodes in your cluster, and
+select a random node each time a request is made.
+
+The CrateDB `Python client`_, `JDBC client`_, and `PHP PDO client`_ support some
+sort of multiple node configuration.
+
+You can implement your own too.
```

Comment:
Again, this sounds vague and not really helpful. You can always implement it yourself :)